Jump_Search/Jump_Search.py

```diff
@@ -0,0 +1,47 @@
+# Python3 code for Jump Search 
+import math 
+
+def jumpSearch( arr , element , number_of_elements ): 
+	
+	# Finding block size to be jumped 
+	step = math.sqrt(number_of_elements) 
```

Comment:
If we type cast here. Is there still need to do so at later stages?

Reply:
Yes here it was for step and down there it was for step =step + sqrt of number of elements

Reply:
Ok 👍